seed/landing/templates/landing/login.html

```diff
@@ -15,9 +15,11 @@ <h2>{% trans "Log in to SEED Platform™" %}</h2>
                     <table class="signup_table">
                         {{form.as_table}}
                     </table>
-                    <input type="submit" class="btn btn-primary" value="{% trans "Log In" %}">
-                    <a href="{% url "landing:landing_page" %}" type="submit" class="btn btn-default">Cancel</a>
-                    <a class="forgot_password" href="{% url "landing:password_reset" %}">{% trans "Forgot password?" %}</a>
+                    <input type="submit" class="btn btn-primary" value="{% trans "Log In" %}"><br>
+                    <div>
+                        <a class="create_account" href="{% url "landing:create_account" %}">{% trans "Create an Account" %}</a>
```

Comment:
Same as before, "Create an Account" is not in lokalise yet but "Create my account" is.